src/Tool/FactoryCreator.php

```diff
@@ -39,6 +56,11 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
 
 EOT;
```

Comment:
Since we're bumping the minimum supported PHP version... PHP 7.3+ allow you to indent heredocs and nowdocs. The closing marker (in this case, `EOT;`) defines the amount of indentation that will be stripped from the start of each line of the heredoc/nowdoc. I've done this in a few other components, and it really helps improve readability.